apps/digital-inclusion/models/save.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const mysql = require('mysql');
+const config = require('../../../config');
+
+const pool = mysql.createPool({
+  host: config.mysql.host,
+  user: config.mysql.user,
+  password: config.mysql.password,
+  database: config.mysql.db
+});
+
+const table = 'research_data';
+
+module.exports = {
+  create(options) {
+    const project = options.project;
+    const researchDate = options.researchDate;
+    const category = options.category;
+    const accessNeeds = options.accessNeeds;
+
+    // build sql statement to deal with empty researchDate or/and accessNeeds,
+    // this will mean a null will go into the db
+    let sql = `INSERT INTO ${table} SET project=?, category=?`;
+    if (researchDate) {
+      sql += ', research_date=?';
+    }
+    if (accessNeeds) {
+     sql += ', access_needs=?';
+    }
+
+    pool.getConnection((err, connection) => {
+    if (err) {
+      // eslint-disable-next-line no-console
+      console.error('error connecting: ' + err.stack);
+    } else {
+      // eslint-disable-next-line no-console
+      console.info('reconnected to the db!');
```

Comment:
Have you re-connected?

Reply:
this is the console log left over from before, I'll change to 
'connect to db or use existing db connection'
Or is that too much information